backend/api/Models/Concepts/ResearchFile/ResearchFilePropertyMap.cs

```diff
@@ -24,6 +24,7 @@ public void Register(TypeAdapterConfig config)
 
             config.NewConfig<ResearchFilePropertyModel, Entity.PimsPropertyResearchFile>()
                 .Map(dest => dest.PropertyResearchFileId, src => src.Id)
+                .Map(dest => dest.Property, src => src.Property)
```

Comment:
@FuriousLlama this is the line I'm not sure about.

Reply:
That looks very much necessary. Not sure why it was not there